### PR TITLE
Reset the entrypoint properly when custom_entrypoint is used

### PR DESCRIPTION
**Pull request by dirkmueller**
the entrypoint needs to be set via --entrypoint not as a command at the end. the command are just the additional parameters when entrypoint is being configured.

### CODE REVIEW

File: pytest_container/container.py [hunk 1]
```diff
@@ -519,24 +518,27 @@ def get_launch_cmd(
         )
 
         id_or_url = self.container_id or self.url
-        bash_launch_end = _CONTAINER_STOPSIGNAL + [
-            "-it",
-            id_or_url,
-            _CONTAINER_ENTRYPOINT,
-        ]
+        cmd.extend(_CONTAINER_STOPSIGNAL)
```
Comment (dcermak) [inline pytest_container/container.py:521]:
```suggestion
```

File: tests/test_container_build.py [hunk 2]
```diff
@@ -35,10 +35,14 @@
...
 )
-#: This is just a busybox container with 4MB of random data in there
+
+#: This is just a busybox container with 4MB of random data
+#: and a custom entrypoint
 BUSYBOX_WITH_GARBAGE = DerivedContainer(
     base=BUSYBOX_WITH_ENTRYPOINT,
-    containerfile="""RUN dd if=/dev/random of=/foobar bs=4M count=1
+    containerfile="""
+ENTRYPOINT /usr/bin/id
```
Comment (dcermak) [inline tests/test_container_build.py:44]: Why did you add the `ENTRYPOINT` here?

Comment (dirkmueller) [reply tests/test_container_build.py]: I've now splitted it into a separate test, which hopefully helps.

**Comment by dirkmueller** [comment]
Its a bit surprising that the docker tests fail. will take a look later.

**Comment by dirkmueller** [comment]
this needs #147  first
